SPECS/xcp-ng-xapi-storage.spec

```diff
@@ -41,19 +68,28 @@ cd build
 #%postun
 #%systemd_postun_with_restart qemuback.service
 
-%files
+%files libs
 %license LICENSE README.md
-#%{_bindir}/qemuback.py
 %{_docdir}/xcp-ng-xapi-storage/
-%{_libexecdir}/xapi-storage-script/
 %{_prefix}/lib/python2.7/site-packages/xapi/storage/libs/
 %{_prefix}/lib/python2.7/site-packages/xcp_ng_xapi_storage_libs-*-py2.7.egg-info
+
+%files datapath-tapdisk
+%{_libexecdir}/xapi-storage-script/datapath/tapdisk
+
+#%files datapath-qemudisk
```

Comment:
Careful with comment containing macros: these will still be evaluated, and sometimes they may be multiline. Better disable the macro with `%%`. Example: `%%files`. Applies to various places in the file.

Reply:
fixed